src/components/IconGithub.jsx

```diff
@@ -1,16 +1,41 @@
+import React, { useState, useEffect } from 'react';
+
 export default function IconGithub() {
+  const [stars, setStars] = useState(0);
+
+  useEffect(() => {
+    const cachedStarCount = localStorage.getItem('starCount');
+    if (cachedStarCount) {
+      setStars(parseInt(cachedStarCount, 10));
+    } else {
+      const fetchData = async () => {
+        try {
+          const { stargazers_count } = await (await fetch(`https://api.github.com/repos/markmead/hyperui`)).json();
```

Comment:
What's with the double `await`?

Reply:
I was trying to condense two lines of code into one. I have updated this and it looks more readable now.